.github/workflows/test.yml

```diff
@@ -0,0 +1,51 @@
+---
+name: Continuous Integration
+"on":
+  - push
```

Comment:
I think, to make CI run if a PR is opened from a fork, that you also need `pull_request` here. (This being a public repo)